gocd/templates/bash/check-cloudbuild.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+/devinfra/scripts/checks/googlecloud/checkcloudbuild.py \
+  --sha="${GO_REVISION_CHARTCUTERIE_REPO}" \
```

Comment:
This variable comes from gocd right, and is generated based on the name of the pipeline group or something?

Reply:
This environment variable corresponds to the material name and is provided by GoCD ([more info](https://docs.gocd.org/current/faq/dev_use_current_revision_in_build.html#:~:text=GO_REVISION_%24%7Bmaterial%20name%20or%20dest%7D)).